<script>
  try {
    const count = 0
    count++
  } catch (error) {
    console.error("some error:", error.message);
  }
</script>
